node-operators/networks/collators/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Overview
+description: Instructions on how to dive in and become a collator in the Moonbeam Network once you are running a node
+---
+
+# Run a Collator on Moonbeam
+
+![Collator Moonbeam Banner](/images/node-operators/networks/collators/collator-banner.png)
+
+## Introduction {: #introduction } 
+
+Collators are members of the network that maintain the parachains they take part in. They run a full node (for both their particular parachain and the relay chain), and they produce the state transition proof for relay chain validators.
+
+There are some [requirements](/node-operators/networks/collators/requirements/){target=blank} that need to be considered before they can become a collator candidate including machine, bonding, and account requirements.
```

Comment:
I think here we should a bit of a bullet list of the requirements to be a collator (like a summary of both technical and "moral"):

- Be engaged with the network and community
- Support the livelyness of the network
- .... some other moral and technical requirements
-

Reply:
What are the other "moral" requirements? Would it be better to add them to the Requirements page?

Reply:
@artkaseman what "moral" requirements should we list here.

We could add them in the requirements page, but I feel that a short summary in the overview would also be helpful